src/sdk/constants/abi/ERC7484RegistryAbi.ts

```diff
@@ -0,0 +1 @@
ress"}],"name":"getDigest","outputs":[{"internalType":"bytes32","name":"digest","type":"bytes32"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"ResolverUID","name":"resolverUID","type":"bytes32"},{"internalType":"address","name":"moduleAddress","type":"address"},{"internalType":"bytes","name":"metadata","type":"bytes"},{"internalType":"bytes","name":"resolverContext","type":"bytes"}],"name":"registerModule","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"contract IExternalResolver","name":"resolver","type":"address"}],"name":"registerResolver","outputs":[{"internalType":"ResolverUID","name":"uid","type":"bytes32"}],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"string","name":"schema","type":"string"},{"internalType":"contract IExternalSchemaValidator","name":"validator","type":"address"}],"name":"registerSchema","outputs":[{"internalType":"SchemaUID","name":"uid","type":"bytes32"}],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"components":[{"internalType":"address","name":"moduleAddress","type":"address"}],"internalType":"struct RevocationRequest[]","name":"requests","type":"tuple[]"}],"name":"revoke","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"attester","type":"address"},{"components":[{"internalType":"address","name":"moduleAddress","type":"address"}],"internalType":"struct RevocationRequest[]","name":"requests","type":"tuple[]"},{"internalType":"bytes","name":"signature","type":"bytes"}],"name":"revoke","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"components":[{"internalType":"address","name":"moduleAddress","type":"address"}],"internalType":"struct RevocationRequest","name":"request","type":"tuple"}],"name":"revoke","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"attester","type":"address"},{"components":[{"internalType":"address","name":"moduleAddress","type":"address"}],"internalType":"struct RevocationRequest","name":"request","type":"tuple"},{"internalType":"bytes","name":"signature","type":"bytes"}],"name":"revoke","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"ResolverUID","name":"uid","type":"bytes32"},{"internalType":"contract IExternalResolver","name":"resolver","type":"address"}],"name":"setResolver","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"ResolverUID","name":"uid","type":"bytes32"},{"internalType":"address","name":"newOwner","type":"address"}],"name":"transferResolverOwnership","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"uint8","name":"threshold","type":"uint8"},{"internalType":"address[]","name":"attesters","type":"address[]"}],"name":"trustAttesters","outputs":[],"stateMutability":"nonpayable","type":"function"}] as const
```

Comment:
Your linter should have complained here 🤔 . Can you force linting over this file? 
`bun run lint --apply-unsafe`

Reply:
done